### PR TITLE
knob to control

### LINKED ISSUE #18499: Update knobs to controls in storybook: ConfirmAddSuggestedToken
### Description
Currently there are a few storybook stories still using knobs in our storybook files `*.stories.js`. knobs have been deprecated in favour of controls. It would be great to migrate these last stories from knobs to controls.

This issue is to migrate the `ui/pages/confirm-add-suggested-token/confirm-add-suggested-token.stories.js` from knobs to controls

Screenshot 2023-04-06 at 4 39 08 PM

### Requirements
If you would like to take on this issue please comment but before asking to take on this issue there are a few requirements that will help complete this task:
- Get storybook up and running `yarn storybook`
- Some knowledge of storybook is not required but would be beneficial for this task. [storybook docs here](https://storybook.js.org/)

### Technical Details
- Migrate knobs to controls

### Acceptance Criteria
- knobs have been migrated to controls for listed story
- Take Before / After screenshots of the knobs and new controls and attach to the PR
- Tag @georgewrmarshall, @garrettbear, or @NidhiKJha in the PR to review

### PR DESCRIPTION
**Pull request by sumitshinde-84**
## Explanation
Replaced the deprecated default prop in Storybook's args with the control prop, which is the recommended way to define controls for Storybook stories.
Updated the symbol and Icon URL knobs to use the control prop instead of the deprecated default prop.
Removed the defaultStory story name, as it is not necessary with the use of the args prop.
Updated the argTypes definition for tokens and suggestedAssets to use the control prop instead of table, as table is not a valid property for argTypes in Storybook.

### Before
before

### After
Screenshot_2023-04-12_03-47-59 ( (1)
knob

## Pre-merge author checklist

- [x] I've clearly explained:
  - [x] What problem this PR is solving
  - [x] How this problem was solved
  - [ ] How reviewers can test my changes
- [ ] Sufficient automated test coverage has been added

## Pre-merge reviewer checklist

- [x] Manual testing (e.g. pull and build branch, run in browser, test code being changed)
- [x] PR is linked to the appropriate GitHub issue
- [ ] **IF** this PR fixes a bug in the release milestone, add this PR to the release milestone

If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.
closes #18499

### CODE REVIEW

File: ui/pages/confirm-add-suggested-token/confirm-add-suggested-token.stories.js [hunk 1]
```diff
@@ -68,11 +77,23 @@ DefaultStory.storyName = 'Default';
 DefaultStory.args = {
   suggestedAssets: [...mockSuggestedAssets],
   tokens: [],
+  symbol: 'META',
+  image: 'metamark.svg',
```
Comment (georgewrmarshall) [inline ui/pages/confirm-add-suggested-token/confirm-add-suggested-token.stories.js:81]: I would remove these and move them to the story config so they are available for all stories
```suggestion
```

File: ui/pages/confirm-add-suggested-token/confirm-add-suggested-token.stories.js [hunk 2]
```diff
@@ -18,15 +17,20 @@ export default {
...
     },
+    symbol: {
+      control: 'text',
+      table: { category: 'Data' },
+    },
+    image: {
+      control: 'text',
+      table: { category: 'Data' },
+    },
   },
```
Comment (georgewrmarshall) [inline ui/pages/confirm-add-suggested-token/confirm-add-suggested-token.stories.js:28]: I think we can add the eth logo here for all stories

```suggestion
  },
  args: {
    symbol: 'ETH',
    image: './images/eth_logo.svg',
  },
```

File: ui/pages/confirm-add-suggested-token/confirm-add-suggested-token.stories.js [hunk 3]
```diff
@@ -84,14 +105,23 @@ WithDuplicateAddress.args = {
       ...mockSuggestedAssets[0].asset,
     },
   ],
+  symbol: 'META',
+  image: 'metamark.svg',
```
Comment (georgewrmarshall) [inline ui/pages/confirm-add-suggested-token/confirm-add-suggested-token.stories.js:109]: Can remove these as they are the same for all stories
```suggestion
```

File: ui/pages/confirm-add-suggested-token/confirm-add-suggested-token.stories.js [hunk 4]
```diff
@@ -104,4 +134,6 @@ WithDuplicateSymbolAndDifferentAddress.args = {
       address: '0xNonSuggestedAddress',
     },
   ],
+  symbol: 'META',
+  image: 'metamark.svg',
```
Comment (georgewrmarshall) [inline ui/pages/confirm-add-suggested-token/confirm-add-suggested-token.stories.js:138]: Can remove these as they are the same for all stories
```suggestion
```

**Review by georgewrmarshall — COMMENTED** [review]
Looking good left some suggestions

**Review by georgewrmarshall — COMMENTED** [review]
Code LGTM! @garrettbear I'm having an issue checking this branch out locally. Have tried a few things. Would you mind doing the visual check?

The error I'm getting
```
error: cannot lock ref 'refs/heads/update': 'refs/heads/update/design-tokens-v1.11.0' exists; cannot create 'refs/heads/update'
From github.com:MetaMask/metamask-extension
 ! [new ref]             refs/pull/18635/head -> update  (unable to update local ref)
exit status 1
```

**Comment by sumitshinde-84** [comment]
> Code LGTM! @garrettbear I'm having an issue checking this branch out locally. Have tried a few things. Would you mind doing the visual check?
> 
> The error I'm getting
> 
> ```
> error: cannot lock ref 'refs/heads/update': 'refs/heads/update/design-tokens-v1.11.0' exists; cannot create 'refs/heads/update'
> From github.com:MetaMask/metamask-extension
>  ! [new ref]             refs/pull/18635/head -> update  (unable to update local ref)
> exit status 1
> ```
This error message indicates that you are trying to create a new branch named "update", but a branch with that name already exists. Specifically, it looks like the branch "refs/heads/update/design-tokens-v1.11.0" already exists, which is preventing you from creating a new branch with the same name.

To resolve this error, you can either delete the existing branch with the name "update/design-tokens-v1.11.0", or you can choose a different name for your new branch.

**Review by georgewrmarshall — CHANGES_REQUESTED** [review]
Looks like the warning message is missing from the `WithDuplicateSymbolAndDifferentAddress`.  You can check the original here https://metamask.github.io/metamask-storybook/?path=/story/pages-confirmaddsuggestedtoken--with-duplicate-symbol-and-different-address

### Before
Screenshot 2023-04-19 at 12 58 35 PM

### After
Screenshot 2023-04-19 at 1 01 24 PM

**Comment by georgewrmarshall** [comment]
> > Code LGTM! @garrettbear I'm having an issue checking this branch out locally. Have tried a few things. Would you mind doing the visual check?
> > The error I'm getting
> > ```
> > error: cannot lock ref 'refs/heads/update': 'refs/heads/update/design-tokens-v1.11.0' exists; cannot create 'refs/heads/update'
> > From github.com:MetaMask/metamask-extension
> >  ! [new ref]             refs/pull/18635/head -> update  (unable to update local ref)
> > exit status 1
> > ```
> 
> This error message indicates that you are trying to create a new branch named "update", but a branch with that name already exists. Specifically, it looks like the branch "refs/heads/update/design-tokens-v1.11.0" already exists, which is preventing you from creating a new branch with the same name.
> 
> To resolve this error, you can either delete the existing branch with the name "update/design-tokens-v1.11.0", or you can choose a different name for your new branch.

Ahh thanks I deleted all branches that started with `update/` and it was able to pull this branch

**Comment by sumitshinde-84** [comment]
> Looks

ohh sorry @georgewrmarshall  i gonna fix it .

**Comment by sumitshinde-84** [comment]
>

**Review by georgewrmarshall — DISMISSED** [review]
Hey @sumitshinde-84, it turns out that this was quite a complex task. I've push an update to use the redux provider instead of what was being used. It should be working now. Apologies for the misleading issue and I appreciate all your effort on this!

**Review by georgewrmarshall — APPROVED** [review]
LGTM! Thanks for the contribution and the git help!

**Review by NidhiKJha — APPROVED** [review]
LGTM! Thanks for your contribution

**Pull request merged**